src/app/pages/TopicPage/TopicTitle/index.jsx

```diff
@@ -17,7 +18,8 @@ import {
 
 const H1 = styled.h1`
   ${({ service }) => getSansBold(service)}
-  margin: ${GEL_SPACING_TRPL}rem 0 ${GEL_SPACING_TRPL}rem 0;
+  color: ${C_GREY_10};
+  margin: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING_TRPL} 0;
```

Comment:
you can use shorthand margin/padding here

```suggestion
  margin: ${GEL_SPACING_TRPL} 0;
```